clients/ui/bff/internal/api/helpers.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 type Envelope[D any, M any] struct {
-	Data     D `json:"data,omitempty"`
+	Data     D `json:"data"`
```

Comment:
@alexcreasy @ederign I've changed this since it was ommiting the `data` field if no data was provided, which I think was misleading, I need to update the api spec though

Reply:
Perfect!